src/library/baseplaylistfeature.cpp

```diff
@@ -455,27 +463,28 @@ void BasePlaylistFeature::slotExportPlaylist() {
     qDebug() << "Export playlist" << playlistName;
 
     QString lastPlaylistDirectory = m_pConfig->getValue(
-            ConfigKey("[Library]", "LastImportExportPlaylistDirectory"),
+            kConfigKeyLastImportExportPlaylistDirectory,
             QStandardPaths::writableLocation(QStandardPaths::MusicLocation));
 
     // Open a dialog to let the user choose the file location for playlist export.
     // The location is set to the last used directory for import/export and the file
     // name to the playlist name.
-    QString filefilter = tr("M3U Playlist (*.m3u)");
-    QString file_location = QFileDialog::getSaveFileName(
+    QString fileFilter = tr("M3U Playlist (*.m3u)");
+    QString fileLocationDlg = QFileDialog::getSaveFileName(
```

Comment:
Confusing naming of variable -> `fileLocationInput`

Reply:
done.